### ISSUE TITLE
Country id Unkown

### ISSUE THREAD

**Issue by Artapel** [yugecin/tsstats#7]
Good day. Do you perhaps have advice on why All users are part of the unknown country

**Comment by yugecin** [comment]
the countries are provided by the teamspeak server, I guess the server can't determine the country.
While in teamspeak, click on a client and see if there's a flag next to their name

**Comment by Artapel** [comment]
Yes the flags for all the users is present. One of my other teamspeak addons also gets the countries correctly

**Comment by kuhberg** [comment]
Can confirm this issue. TeamSpeak is showing everything correctly.
